Buildroot patch for pru-software-support package to disable building examples
and instead build lab5.

Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@gmail.com>
---

--- a/package/pru-software-support/pru-software-support.mk	2021-03-06 23:16:45.000000000 +0200
+++ b/package/pru-software-support/pru-software-support.mk	2021-05-04 23:09:56.477162512 +0300
@@ -26,7 +26,7 @@ define HOST_PRU_SOFTWARE_SUPPORT_INSTALL
 endef
 
 define PRU_SOFTWARE_SUPPORT_BUILD_CMDS
-	$(MAKE) PRU_CGT=$(TI_CGT_PRU_INSTALLDIR) -C $(@D)/examples
+	$(MAKE) PRU_CGT=$(TI_CGT_PRU_INSTALLDIR) -C $(@D)/labs/Hands_on_Labs/lab_5/solution/am335x/PRU_RPMsg_Echo_Interrupt1
 endef
 
 define PRU_SOFTWARE_SUPPORT_INSTALL_TARGET_CMDS
